data/campaigns/Winds_of_Fate/scenarios/02x_The_Feast.cfg

```diff
@@ -0,0 +1,143 @@
+#textdomain wesnoth-wof
+
+[scenario]
+    id=02x_The_Feast
+    name= _ "The Feast"
+    next_scenario=03_The_Contention
+    {NO_MAP_DATA}
+    carryover_percentage=0
+
+    {INTRO_AND_SCENARIO_MUSIC heroes_rite.ogg ()}
+
+    [story]
+        [part]
+            story= _ "Fresh prey carcasses were justly divided among Hunters in accord with the Ways of Morogor. An ample victory feast was had..."
+        [/part]
+        [part]
+            background={KARRON_PORTRAIT}
+            scale_background=no
+            story={CAPTION ( _ "Karron")} + _ "Savory...  (<i>crunch</i>)"
+        [/part]
+        [part]
+            background={KARRON_PORTRAIT}
+            scale_background=no
+            story={CAPTION ( _ "Karron")} + _ "Sapid...  (<i>chewing</i>)"
+        [/part]
+        [part]
+            background={KARRON_PORTRAIT}
+            scale_background=no
+            story={CAPTION ( _ "Karron")} + _ "...these creatures are a delicacy without match."
+        [/part]
+        [part]
+            background={RESHAN_PORTRAIT}
+            scale_background=no
+            story={CAPTION ( _ "Reshan")} + _ "Gorlack, I bear rich tidings.
+The captive infester’s mind spilled every secret it held."
+        [/part]
+        [part]
+            background={GORLACK_PORTRAIT}
+            scale_background=no
+            story={CAPTION ( _ "Gorlack")} + _ "Excellent, Reshan! Speak of it."
+        [/part]
+        [part]
+            background={RESHAN_PORTRAIT}
+            scale_background=no
+            story={CAPTION ( _ "Reshan")} + _ "‘Orc’ the creatures call themselves."
+        [/part]
+        [part]
+            background={KARRON_PORTRAIT}
+            scale_background=no
+            story={CAPTION ( _ "Karron")} + _ "(<i>examining his meal</i>)
+
+... ' Orkchops. '"
```

Comment:
Mmm, tasty orcs! If it wasn't obvious before that drakes feast on sentient races (recalling discussion from past revisions) it's pretty clear here!

Reply:
After the events of TRoW, the orcs receive their just *des**s**erts*.